src/v2/guide/instance.md

```diff
@@ -16,46 +16,68 @@ var vm = new Vue({
 
 Although not strictly associated with the [MVVM pattern](https://en.wikipedia.org/wiki/Model_View_ViewModel), Vue's design was partly inspired by it. As a convention, we often use the variable `vm` (short for ViewModel) to refer to our Vue instance.
 
-When you instantiate a Vue instance, you need to pass in an **options object** which can contain options for data, template, element to mount on, methods, lifecycle callbacks, and more. The full list of options can be found in the [API reference](../api/#Options-Data).
+When you create a Vue instance, you pass in an **options object**. The majority of this guide describes how you can use these options to create your desired behavior. For reference, you can also browse the full list of options in the [API reference](../api/#Options-Data).
 
-The `Vue` constructor can be extended to create reusable **component constructors** with pre-defined options:
+A Vue application consists of a **root Vue instance** created with `new Vue`, optionally organized into a tree of nested, reusable components. For example, a todo app's component tree might look like this:
 
-``` js
-var MyComponent = Vue.extend({
-  // extension options
-})
-
-// all instances of `MyComponent` are created with
-// the pre-defined extension options
-var myComponentInstance = new MyComponent()
+```
+Root Instance
+|- TodoList
+   |- TodoItem
+      |- DeleteTodoButton
+      |- EditTodoButton
+   |- TodoListFooter
+      |- ClearTodosButton
+      |- TodoListStatistics
```

Comment:
Thanks for bringing this to my attention! I'll do a little more user testing on this part.